crmprtd/__init__.py

```diff
@@ -101,6 +91,14 @@ def common_script_arguments(parser):    # pragma: no cover
                               'observations')
     parser.add_argument('-i', '--input_file',
                         help='Input file to process')
+    parser.add_argument('--chunk_size', type=int,
+                        default=4096,
```

Comment:
Ahhhh, so I finally had an epiphany about why the chunking+bisecting is faster than bisecting alone and that's because you use a power of two for the base chunk size. I *bet* that we can get comparable performance without the need for configuration by breaking the observation set into chunks comprised of maximal powers of two. E.g. if you have 1200 observations, break it up into (1024, 128, 32, 16). I think we're running out of time, so we should relegate this to future work, but I *bet* that it would be the fastest.